doc/TRY.md

```diff
@@ -57,7 +57,11 @@ running a too old glibc. In this case you may have to build from source. See the
 about this in the README](../README.md#a-note-on-linux-binaries-and-glibc-version).
 
 For every file available on the website, there is an accompanying `.asc` file with the same
-name on our [Github release page](https://github.com/wizardsardine/liana/releases). This is a GPG signature made with Antoine Poinsot's key:
+name on our [Github release page](https://github.com/wizardsardine/liana/releases).
+
+if Liana version is inferior to v7:
+
+This is a GPG signature made with Antoine Poinsot's key:
```

Comment:
Probably not worth giving these potentially confusing details, as user would never want to run an earlier version, and certainly not in the TRY.md.